internal/audit/server_checks.go

```diff
@@ -230,3 +230,48 @@ func makeCheckServerResourceLimits(memoryUsageThreshold, storeUsageThreshold flo
 		return Pass, nil
 	}
 }
+
+func checkServerLeafnodeNames() checkFunc {
+	return func(r *archive.Reader, examples *ExamplesCollection) (Outcome, error) {
+
+		checkIfNameContainsSpaces := func(name string) bool {
```

Comment:
this should probably check for the entire class of unicode 'whitespace', not just a space.

Reply:
while I agree, for the purpose of the case at hand only actual space would be a problem - other unicode space-like characters are fine

Reply:
I was thinking of newlines, tabs, etc. `unicode.IsSpace` does this.